dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/Unpickler.cs

```diff
@@ -28,27 +20,12 @@ namespace Razorvine.Pickle
 [SuppressMessage("ReSharper", "InvertIf")]
 public class Unpickler : IDisposable {
 
-	protected const int HIGHEST_PROTOCOL = 4;
+	protected internal const int HIGHEST_PROTOCOL = 4;
```

Comment:
actually, it was my intention to expose this value. User code should be able to read it (also Pickler.HIGHEST_PROTOCOL). I think it was my mistake to make it protected here, and it actually should have been  public. I think I'll change this in another commit.

Reply:
I made it public